@xen-orchestra/lite/package.json

```diff
@@ -21,6 +21,7 @@
     "complex-matcher": "^0.7.0",
     "json-rpc-2.0": "^1.3.0",
     "pinia": "^2.0.14",
+    "placement.js": "^1.0.0-beta.5",
```

Comment:
It's not "dangerous" to use a lib in beta for production use ?
Some features may not be stable.

Reply:
Since the lib is used for a specific usage, and this specific usage works correctly, I don't think there is any risk here.